package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
There shouldn't be a package.json file in the root directory (when mono-repo causes confusion 😅)